jest.tz.unit.conf.json

```diff
@@ -3,8 +3,12 @@
     "\\.(css|less)$": "<rootDir>/frontend/test/__mocks__/styleMock.js",
     "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/frontend/test/__mocks__/fileMock.js",
     "^promise-loader\\?global\\!metabase-lib\\/metadata\\/utils\\/ga-metadata$": "<rootDir>/frontend/src/metabase-lib/metadata/utils/ga-metadata.js",
-    "^d3-(.*)$": "<rootDir>/node_modules/d3-$1/dist/d3-$1"
+    "^d3-(.*)$": "<rootDir>/node_modules/d3-$1/dist/d3-$1",
+    "react-markdown": "<rootDir>/node_modules/react-markdown/react-markdown.min.js"
```

Comment:
This line is needed since I've encountered an error after upgrading `react-markdown` to v8.0.4.

Ref: https://stackoverflow.com/questions/72382316/jest-encountered-an-unexpected-token-react-markdown